package/package.json

```diff
@@ -60,7 +60,7 @@
         "@kusto/language-service-next": "0.0.54"
     },
     "peerDependencies": {
-        "monaco-editor": "0.33.0"
```

Comment:
Does this need to be a specific version?

Can we just do 0.X.X? Or whatever the valid equivalent is?

Reply:
Hm. I guess it can be. The up-upstream is using specific, but that doesn't really mean anything: https://github.com/Azure/monaco-kusto/blob/master/package/package.json#L61.

Change it?

Reply:
If you think of it next time you make a change.

I also found out `package.json` has an `overrides` feature which can help so we don't need to use `--legacy-peer-deps` all the time.